tests/testthat/test-rvar-apply.R

```diff
@@ -42,6 +58,31 @@ test_that("rvar_apply works", {
   ref <- x
   ref["a1",,] <- ref["a1",,] + 1
   ref["a2",,] <- 0
-  rvar_apply(x, 1, function(x) if(mean(draws_of(x)) > 18) rvar(0) else x + 1)
+  expect_equal(rvar_apply(x, 1, function(x) if(mean(draws_of(x)) > 18) rvar(0) else x + 1), ref)
 })
 
+
+# rvar_apply with factors -------------------------------------------------
+
+test_that("rvar_apply works with rvar_factor", {
```

Comment:
Is there reason to believe this could work differently with `rvar_ordered`? In other words, would it make sense to test `rvar_ordered` variables here too?

Reply:
Good idea. Adding a simple test (not a whole new test case) to check if the ordered property is preserved (if it works in one sub-case it should in the other)